docs/catalog/ai/right-hardware-type.md

```diff
@@ -0,0 +1,37 @@
+---
+version: 1.0
+submitted_by: navveenb
+published_date: tbd
+category: ai
+tags: 
+ - ai
+ - role:data-scientist
```

Comment:
Should we add `machine-learning` as a tag here as well since it is being referenced in the pattern content?